common/templates/emails/sync_learner_profile_data_email_change_body.txt

```diff
@@ -0,0 +1,25 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers %>
+
+<p>
+${_("The email associated with your {platform_name} account has changed from {old_email} to {new_email}.").format(
+    platform_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
+    old_email=old_email,
+    new_email=new_email,
+)}
+</p>
+
+
+<p>${_("No action is needed on your part.")}</p>
+
+<p>
+${_("If this change is not correct, contact {link_start}{platform_name} Support{link_end} "
+    "or your administrator.").format(
+        platform_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
```

Comment:
@UmanShahzad Could you extract `configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)` into a variable, to avoid performing the lookup twice?

Reply:
Yes, sure! :+1: